### PR TITLE
Fix bug in view (q)settings handling

### PR DESCRIPTION
**Pull request by joernu76**

### CODE REVIEW

File: mslib/msui/mpl_qtwidget.py [hunk 1]
```diff
@@ -56,12 +56,51 @@
...
+    "colour_ft_vertices": (0, 0, 1, 1),
+    "colour_ft_waypoints": (1, 0, 0, 1)}
+
+_DEFAULT_SETTINGS_SIDEVIEW = {
+    "vertical_extent": (1050, 180),
+    "vertical_axis": "pressure",
+    "secondary_axis": "no secondary axis",
+    "plot_title_size": "default",
+    "axes_label_size": "default",
+    "flightlevels": [],
```
Comment (ReimarBauer) [inline mslib/msui/mpl_qtwidget.py]: an old entry was:
https://github.com/Open-MSS/MSS/pull/1705/files#diff-46d3dceb21b8fa62a87ac72721c0c16241048c1eef22ecc21fa3786f282f8f6bL68
"flightlevels": [300, 320, 340]

I'm not verified if that was used, just seen in the review

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
At least using sideview is a problem on clicking options - ok

options_problem

**Comment by joernu76** [comment]
> At least using sideview is a problem on clicking options - ok
> 
> options_problem [ options_problem ](https://user-images.githubusercontent.com/278728/226360325-e5b9737d-8684-4b40-a5fb-548971decc4b.gif) [ ](https://user-images.githubusercontent.com/278728/226360325-e5b9737d-8684-4b40-a5fb-548971decc4b.gif)

Fixed this and some other issues in linear view.

File: tests/_test_msui/test_sideview.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
-# -*- coding: utf-8 -*-
+# -*- coding: utf-7 -*-
```
Comment (ReimarBauer) [inline tests/_test_msui/test_sideview.py]: typo?

UTF-7 ist not part of the unicode standard

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
unicode standard? -8 ?

tests/_test_msui/test_sideview.py

**Comment by ReimarBauer** [comment]
found a different problem. The stored draw flight track color is not used for the initialization on a restart in the Sideview
also other flight track colors are not used
color_options_not_used


After clicking options and ok, they are updated
option_color_not_used

in Topview this is doing as expected

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
see comment example, sideview is not using stored settings on restart

**Review by ReimarBauer — APPROVED** [review]
Thx

**Pull request merged**